### ISSUE TITLE
This is better than my ham-fisted attempt to compare algos.  

### ISSUE THREAD

**Issue by salamanders** [levyben/DeepSuperLearner#2]
I tried a brute force approach a year ago (and pre interest in DL) 

* https://www.reddit.com/r/MachineLearning/comments/4ftvnr/q_eli5_why_not_have_a_royal_rumble_between_all/
* https://ml-allthethings.firebaseapp.com/

... but yours seems much better.

Can you run yours against a good sized number of classic datasets and see how it compares?

**Comment by levyben** [comment]
My plans are to add more validation of performance, you are more than welcomed to do comparisons to classic datasets and upload it here (I'll add it to the repo).
